roles/setup/tasks/tls.yml

```diff
@@ -6,7 +6,7 @@
     mode: '0750'
     owner: "{{ receptor_user }}"
     group: "{{ receptor_group }}"
-    recurse: true
+    recurse: false
```

Comment:
with this set to `true` I was getting "changed" status for TLS files, even if the files were the same.

I think recurse only sets permissions, but we do that in another task anyways so shouldn't be needed here.

Reply:
Instead of setting it to false, just take the param off as false is the default?